ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -801,197 +368,113 @@ sub update_afs_cells ( $$ ) {
-sub Unconfigure_PAM {
-##########################################################################
-  my ($self,$config)=@_;
-  my $libpam;
-  my $libpam_options_auth;
-  my $libpam_options_refresh;
-  my $ret = 0;
+activates/deactivates the component.
 
-  if ($OSname =~ /Solaris/) {
-    $self->warn("Somebody needs to write something to unconfigure PAM on Solaris"); #FIXME
-    return 0;
+=item /software/components/afsclt/thiscell : string
 
-    my $pam_config = '/etc/pam.conf';
-
-
-  } elsif ($OSname =~ /Linux/) {
-
-    my $pam_systemauth = "/etc/pam.d/system-auth";
-    my $pam_screensaver = '/etc/pam.d/screensaver-auth';
-
-    $ret = LC::Check::file($pam_systemauth,
-			   backup => '.ncm_orig',
-			   source => $pam_systemauth,
-			   code => sub {  # gets actual, returns expected.
-				    my $actual = shift;
-				    my @actual = split (/\n/, $actual);
-				    my @expected = grep { $_ !~
-							   /(added by $compname|pam_afs|pam_krb5afs|pam_heimdal)/
-							 } @actual; # remove other AFS/Krb PAMs
-				    my $expected = join("\n",@expected)."\n";
-				    $self->debug(5, "$pam_systemauth should be:\n$expected");
-				    return $expected;
-				   }
-			  );
-
-    $ret = LC::Check::file($pam_screensaver,
-			   backup => '.ncm_orig',
-			   source => $pam_screensaver,
-			   code => sub {  # gets actual, returns expected.
-				    my $actual = shift;
-				    my @actual = split (/\n/, $actual);
-				    my @expected = grep { $_ !~
-							   /(added by $compname|pam_afs|pam_krb5afs|pam_heimdal)/
-							 } @actual; # remove other AFS/Krb PAMs
-				    my $expected = join("\n",@expected)."\n";
-				    $self->debug(5, "$pam_screensaver should be:\n$expected");
-				    return $expected;
-				   }
-			  );
+local AFS cell for this machine
 
-  } else {
-    $self->warn("Don't know how to unconfigure PAM on $OSname");
-    return 0;
-  }
+=item /software/components/afsclt/thesecells : list
 
-  return $ret;
+cell list to authenticate to
 
+=item /software/components/afsclt/verbose : boolean
 
-}
+lots of debugging output in the logs.
 
+=item /software/components/afsclt/cachesize : int
 
-1; #required for Perl modules
+desired AFS cache size on disk, in 1K blocks. The running AFS cache
+will get adjusted online, and $afs_cacheinfo will be changed if
+required. Please note that an available (mounted) AFS cache partition
+has precedence over this value, i.e. you cannot force a lower usage of
+the cache partition. For Linux machines, a cache partition will use
+CACHESIZE=AUTOMATIC, for other OSes, a hardcoded fill rate of 85% is
+used.
+
+=item /software/components/afsclt/cellservdb : string
+
+A regularly-updated AFS CellServDB URL or filename (e.g. from AFS)
+that this component will copy to local disk. The local AFS client will
+get notified of any additions or changes within a cell.
+
+=item /software/components/afsclt/afsd_args : nlist
+
+nlist of arguments to stash in /etc/afsd.args used by rc.afs.
+
+=back
```

Comment:
delete this stuff, just make sure that jaroslav is in the authors in the pom.xml (and maybe add yourself as maintainer?)